### ISSUE TITLE
Settings navigation out of order when using talkback

### ISSUE THREAD

**Issue by soupslurpr** [soupslurpr/BeauTyXT#47]
When using talkback, going to the next item after the pitch black background setting is the back button and the Settings title, and then it goes to the other options. 

I don't know why this is happening and everything I have tried hasn't worked.
